doc/Glossary/index.md

```diff
@@ -5,74 +5,198 @@ permalink: /:path/
-| Transport channel                                       | A Kaa client component responsible for communication with a Kaa server. Refer to Transports for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
```

Comment:
The Kaa platform comprises [Kaa server]({{root_url}}Glossary/#kaa-server) and [endpoint SDKs]({{root_url}}Glossary/#endpoint-sdk).